BitbucketServerAutomation/Functions/Invoke-BBServerPullRequestMerge.ps1

```diff
@@ -0,0 +1,39 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Invoke-BBServerPullRequestMerge

```

Comment:
Rename this function to `Merge-BBServerPullRequest`.

Add comments for these parameters.

Explain what the `$Version` parameter is.

Parameters should always have the first letter upper case, except when the parameter name is less than two characters. Then it is all caps.

Turn on strict mode.

Add `Use-CallerPreference`.